backend/file/file.go

```diff
@@ -86,6 +86,10 @@ func (f *Wrapper) GetMonitorChannel() types.MonitorChannel {
 	return nil
 }
 
+func (f *Wrapper) GetCloneStatus() (string, error) {
```

Comment:
Why is error a string and not a number/enum?

Reply:
error is a built in type, but custom error can be returned, that will contains the actual error

Reply:
Ok..